test/uk/gov/hmrc/cbcr/services/RetrieveReportingEntityServiceSpec.scala

```diff
@@ -54,11 +54,13 @@ class RetrieveReportingEntityServiceSpec extends LogCapturing with UnitSpec with
       isRetrieveReportingEntityTrue(retrieveReportingEntityServiceWithMockRunMode) shouldBe false
     }
 
-    "log info if the configuration value does not match any of the required values" in {
-      withCaptureOfLoggingFrom(Logger) { logs =>
-        new RetrieveReportingEntityService(reportingEntityDataRepo, configuration, mockRunMode, audit)
-        when(mockRunMode.env) thenReturn "wrongEnv"
-      }
-    }
+//    "log info if the configuration value does not match any of the required values" in {
```

Comment:
maybe let's put an ignore for time being or is any way to be fixed?

Reply:
Agreed to leave it as is